exchanges/graphcache/src/store/store.ts

```diff
@@ -154,11 +155,39 @@ export class Store implements Cache {
     invalidate(this, createRequest(query, variables));
   }
 
+  invalidateEntity(entity: Data | string) {
+    const entityKey =
+      typeof entity === 'string' ? entity : this.keyOfEntity(entity);
+
+    if (!entityKey) {
+      invariant(
+        entityKey,
+        `Can't generate a key for invalidateEntity(${entity}),
```

Comment:
This adds an empty indentation on the next line. Let’s maybe keep this in line with the other messages and just manually concatenation?

Also let’s maybe make sure that this message is as clear as this one? https://github.com/FormidableLabs/urql/blob/master/docs/graphcache/help.md#7-cant-generate-a-key-for-readfragment-

The text can be copied almost 1:1, but that one accounts for edge cases, like talking about “keys” in general instead of the default “id” fields